Pod/Classes/AFMInfoBanner.m

```diff
@@ -228,6 +249,18 @@ - (void)layoutSubviews
     [super layoutSubviews];
 }
 
+- (void)updateUI
+{
+    // First pass calculates the height correctly with existing constraints.
+    // Self-only doesn't calculate height on iOS 6, so pass through a superview
```

Comment:
Do we actually still support iOS 6 in there? Shouldn't we drop it in future updates?